code/game/gamemodes/cult/items/talisman.dm

```diff
@@ -17,7 +18,10 @@
 /obj/item/paper/talisman/examine(mob/user)
 	. = ..()
 	if(iscultist(user) && rune)
-		to_chat(user, "The spell inscription reads: <span class='cult'><b><i>[rune.name]</i></b></span>.")
+		var/network_text = ""
+		if(network)
+			network_text = " This spell's network tag reads: <span class='cult'><b><i>[network]</i></b></span>."
+		to_chat(user, "The spell inscription reads: <span class='cult'><b><i>[rune.name]</i></b></span>.[network_text]")
```

Comment:
Use SPAN_CULT here

Reply:
I'm glad that exists, looking at the repetitive markup was hurting me.